account_invoice_report_hls/report/invoice_delivery_report.py

```diff
@@ -31,6 +32,9 @@ def _get_report_base_filename(self):
         self.ensure_one()
         return self.invoice_id._get_report_base_filename()
 
+    def _get_sorted_lines(self):
+        return self.line_ids.sorted(key=lambda r: (r.date_delivered))
+
```

Comment:
We may actually delete this and just add `_order = "date_delivered"` to the report line model.